tests/device_tests/test_msg_recoverydevice_slip39_advanced_dryrun.py

```diff
@@ -65,7 +65,7 @@ def input_flow():
     )
 
 
-@pytest.mark.setup_client(mnemonic=MNEMONIC_SLIP39_ADVANCED_20, passphrase=True)
+@pytest.mark.setup_client(mnemonic=MNEMONIC_SLIP39_ADVANCED_20, passphrase="TREZOR")
```

Comment:
is `passphrase` even used here?

Reply:
Apparently not, the test works even when deleting the `passphrase` argument

Reply:
Fixed in 40530e8